### ISSUE TITLE
Please update mellowplayer to v. 3.6.6

### ISSUE THREAD

**Issue by gorkawien** [void-linux/void-packages#25415]
In order to correct some streaming services that stop working, and update is necessary.

https://bintray.com/colinduquesnoy/MellowPlayer/Stable/3.6.6

**Comment by fosslinux** [comment]
```
<!-- Don't request update of package. We have a script for that. https://alpha.de.repo.voidlinux.org/void-updates/void-updates.txt . However, a quality pull request may help. -->
```

**Comment by abenson** [comment]
Closed by 1775e0692c3912ff4291766eb01ce2d4955afbb6 /  #25709
